src/workloads/query/ProjectParse.yml

```diff
@@ -1,13 +1,21 @@
 SchemaVersion: 2018-07-01
 Owner: "@mongodb/query"
 Description: |
-  Test $project parsing performance for small and large number of fields.
+  Test $project parsing performance for small and large number of fields. We previously had a quadratic
```

Comment:
Could you put a date in here to go with the "previously". Maybe a parenthetical "(before August 2023)"?